### ISSUE TITLE
Client output is messy

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#84]
For example:

```
loom upload file *
Skipping directory tmp

Enter a complete description of the data source. Provide enough detail to ensure traceability.
>

Uploading a.txt ...
Created file source record a.txt@bae39e78558e3b54c19191bdc7a832eeb75e2c8daf48def9cad8aa4b5baa6ca4
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'197fc295f1c255ef3886b5f419100c689d1c1a5643eec234b04190717cde7c70', u'hash_value': u'9a6a9c9074509fbff3a65e819bb7eb7f', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_bae39e7855_a.txt'}
Uploading a.txt~ ...
Created file source record a.txt~@c993b0dcd6e5d2e72c88f5a76a57fbcf091f6b2002128948aee6cd5df0b4f2b4
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'8ab01b68209c7d9d4c7f896054a5ea5a98ee161b700d2e316de9dede6353d99e', u'hash_value': u'686774b23fd835b92cee737b6e027466', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_c993b0dcd6_a.txt~'}
Uploading b.txt ...
Created file source record b.txt@a71a1c23e67c1fff987764e9e49c0ccd9483fa5361e0f93f5b35e42b3ecb8046
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'5d43066f8c8db44346bf75abec8d3c68cda8eeb46330870f4dddb7ad8b0c5431', u'hash_value': u'3b5d5c3712955042212316173ccf37be', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_a71a1c23e6_b.txt'}
Uploading c.txt ...
Created file source record c.txt@782747048d376c3e3341d5fff418edcf22bb4a78dc0a6c96e75429d9d87e3374
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'c6a6360c009d044d7b1e98e223134e11da2b2a946b30d812e167f0ca340ab783', u'hash_value': u'2cd6ee2c70b0bde53fbe6cac3c8b8bb1', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_782747048d_c.txt'}
Uploading first.txt ...
Created file source record first.txt@c3551bfa5da05de38f80d0d58775fce6c9e31eab9a34b0f3cb523df991aba0e2
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'3166e42e52288f8741d30f2a21d1d2f50c317a618cc2283cddd517435e6ee49f', u'hash_value': u'5bbf5a52328e7439ae6e719dfe712200', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_c3551bfa5d_first.txt'}
Uploading hello.txt ...
Created file source record hello.txt@8dba27c0148914c2a90de966ec0e336d30dfb978547ce0fe603eca62d8481af5
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'5db9047ed717ef4a8316657e1e15d066bedf4768afafc7059997bbfd91e45b21', u'hash_value': u'881d684f890752d024b01743c7c2757a', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_8dba27c014_hello.txt'}
Uploading hello.txt~ ...
Created file source record hello.txt~@00d2784a20eff6cb4db639c1975bee43f09ebf77da325d00f813d3b0a20218a4
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'a7a67818515a51d0e6b1bb638bf692c251c086984e72169dd1a48368b5da5224', u'hash_value': u'261a99a433c142f458a89debf3cd5010', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_00d2784a20_hello.txt~'}
Uploading hello_world.json ...
Created file source record hello_world.json@2d9ee2f8892412d78fa307e78eaffa3987260106f525835c2f7f3db4774a14cb
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'b73cdbc7d845e22a72160cfbe4c42638c7390e65588d75f8c01e81e40f5dca21', u'hash_value': u'ecf493d34fdcce77cdeacb27908e34a4', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_2d9ee2f889_hello_world.json'}
Uploading hello_world.json~ ...
Created file source record hello_world.json~@152fd2a820c7aeced64489b99ca8f2b91dde3ff35071c9445ea54b0460e117af
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'3f17316c9e51ec0b4d3aca8832ff93a97ca639fd6fbbe338213f9e4c549c2f4f', u'hash_value': u'611ab469a9047de3f1b457ea617e5da8', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_152fd2a820_hello_world.json~'}
Uploading inputs.json ...
Created file source record inputs.json@94bb9b80e1c5fd916e340051a6f60e696ec61b893ee1a19016b6aa7ea8bd74a1
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'6fa97128829c16d744925f6cdc15d494d7994c7356690fc993ca5437e4d3bc67', u'hash_value': u'c443cabaf1d4d660942de903d6a03134', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_94bb9b80e1_inputs.json'}
Uploading second.txt ...
Created file source record second.txt@dc36dcf4a693d4e3e73a7026f1bfd9b8642b7e8bad36d4dbc14ad87cc89a1f64
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'a1c4d3259c3a3fd4a10431a7a1a842c806c5815df03c1afc44ea481a9ca4a2d2', u'hash_value': u'c193497a1a06b2c72230e6146ff47080', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_dc36dcf4a6_second.txt'}
Uploading workflow.json ...
Created file source record workflow.json@215246a53fbb26e034ab3221e965a5afc1a052cee5683bf84e618458eaca1903
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'b1814ba835de52141119889544c841bdf86ee54aa8f2a35d693c76cc73501683', u'hash_value': u'bb14274096846cfde68461d332b16275', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_215246a53f_workflow.json'}
Uploading workflow_run.json ...
Created file source record workflow_run.json@d844363519699bebeebf6d75cb8d66f8461d7e791621d594544269acb794773a
Uploading to destination location {'host_url': u'localhost', 'file_contents': {u'_id': u'1977432c47124b708c2629267c86cc14e0da7925df7035958b7ff5c52aa8ce38', u'hash_value': u'98ebcc11c4dce842779216b8ee089208', u'hash_function': u'md5'}, 'file_path': u'/Users/nhammond/working_dir/imported_files/2016-05-05T23:10:06.631642+00:00_d844363519_workflow_run.json'}
```

**Comment by nhammond** [comment]
Also, line break on prompts looks strange:
```
prompt goes here
>
answer goes here
```

**Comment by nhammond** [comment]
Fixed in 32d7584ec5040f4
